src/check/rosetta_installed.rs

```diff
@@ -0,0 +1,62 @@
+use nix_rs::{
+    env::{MacOSArch, OS},
+    info,
+};
+use serde::{Deserialize, Serialize};
+use std::process::Command;
+
+use crate::traits::{Check, CheckResult, Checkable};
+
+/// Check if rosetta 2 is installed
+///
+/// Applies to ARM macs.
+#[derive(Debug, Default, Serialize, Deserialize, Clone)]
+#[serde(default)]
+pub struct RosettaInstalled {
+    enable: bool,
+    required: bool,
+}
+
+impl Checkable for RosettaInstalled {
+    fn check(
+        &self,
+        nix_info: &info::NixInfo,
+        _: Option<&nix_rs::flake::url::FlakeUrl>,
+    ) -> Vec<Check> {
+        let mut checks = vec![];
+        if let (
+            true,
+            OS::MacOS {
+                nix_darwin: _,
+                arch: MacOSArch::Arm64(_),
+            },
+        ) = (self.enable, &nix_info.nix_env.os)
+        {
+            let check = Check {
+                title: "Rosetta Installed".to_string(),
+                info: "".to_string(),
```

Comment:
This should be something rather than empty string, so that we don't see an empty line (somewhat confusing) here:

<img width="216" alt="image" src="https://github.com/user-attachments/assets/16854dc2-bff7-42a5-81f2-65fc4dd5afb7">